widgetssdk/src/main/java/com/glia/widgets/core/secureconversations/domain/GetUnreadMessagesCountWithTimeoutUseCase.kt

```diff
@@ -0,0 +1,39 @@
+package com.glia.widgets.core.secureconversations.domain
+
+import androidx.annotation.VisibleForTesting
+import com.glia.widgets.core.secureconversations.SecureConversationsRepository
+import com.glia.widgets.helper.Logger
+import io.reactivex.Single
+import java.util.concurrent.TimeUnit
+
+private const val TAG = "GetUnreadMessagesCountUseCase"
+
+/**
+ * The timeout for
```

Comment:
Looks like the sentence is not finished here: "The timeout for"

Reply:
fixed 3c335379e0642ec393d2785ee787c928665e5d0b